### PR TITLE
My Portofolio project

### PR DESCRIPTION
**Pull request by sarazaToussaint**
In this project we :

- Add Makeup for the portfolio project
- Add stylesheet for the portfolio project
- Use linters i

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+            <h2>Contact me</h2>
+            <p>
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I’d
+              love to help with it
+            </p>
+          </div>
+
+          <div class="form-field">
+            <form action="/my-handling-form-page" method="post">
```
Comment (micheaol) [inline index.html:235]: Please ensure that the `form` action value is similar to `https://formspree.io/f/{form_id}` according to the project requirement.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+                    required
+                  />
+                </li>
+                <li>
+                  <textarea
+                    id="msg"
+                    name="user_message"
+                    maxlength="500"
+                    placeholder="Write your message here"
+                  ></textarea>
```
Comment (micheaol) [inline index.html:262]: Please ensure that all input fields have the `required` attribute to activate the validation on submit.

File: style.css [hunk 3]
```diff
@@ -0,0 +1,383 @@
...
+  color: #172b4d;
+  font-weight: bold;
+  line-height: 60px;
+}
+
+.contact-form p {
+  color: #fff;
+  height: 102px;
+  padding: 0;
+  top: 4802px;
```
Comment (micheaol) [inline style.css:72]: Please ensure that flexbox is used to layout the elements of the `form`.

File: style.css [hunk 4]
```diff
@@ -0,0 +1,383 @@
...
+  line-height: 52px;
+}
+
+.introduction_section {
+  background-image: url("./images/header-shapes\ mobile.svg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  padding: 50px 30px 150px 30px;
+  background-color: white;
+  border-bottom-left-radius: 100px;
```
Comment (micheaol) [inline style.css:57]: Please ensure that you stick to the design as much as possible (e.g., font, colors, images, tests, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your Implementation | Figma Design
:---:|:---:
image | image

File: index.html [hunk 5]
```diff
@@ -0,0 +1,274 @@
...
+                </li>
+                <li>
+                  <button type="submit">Get in touch</button>
```
Comment (micheaol) [inline index.html:265]: Please Implement the button interactions (enable, hover, etc.) according to the project requirement.

**Review by micheaol — CHANGES_REQUESTED** [review]
Hi @sarazaToussaint , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## TO HIGHLIGHT: 
- Your project is professional 💯 
- Your repo is professional 💯 
- Well structured files 💯 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 6]
```diff
@@ -0,0 +1,244 @@
...
+            <h2>Contact me</h2>
+            <p>
+              If you have an application you are interested in developing, a
+              feature that you need built or a project that needs coding. I’d
+              love to help with it
+            </p>
+          </div>
+
+          <div class="my_form">
+            <form action="https://formspree.io/f/form_id" method="post" class="form_content">
```
Comment (VICTOR-CHUKWU) [inline index.html:234]: - [ ] Nice work so far @sarazaToussaint , currently your form is not being submitted as it returns a 404 error, I think you should create an account with formspree. you can use this [link](https://formspree.io/login)
                   
image

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @sarazaToussaint 👋  

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 6 above)
Comment (sarazaToussaint) [reply index.html]: Thank you @VICTOR-CHUKWU, now it done

**Review by Deepakdanger — APPROVED** [review]
Hi @sarazaToussaint, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**